ACPI/SSDT-Xhci.asl

```diff
@@ -60,77 +76,81 @@ DefinitionBlock ("", "SSDT", 2, "OSY86 ", "Xhci", 0x00001000)
 
         Scope (RHUB)
         {
-            Scope (HS01) // front charging
+            Scope (HS01)
             {
                 Method (_DSM, 4, NotSerialized)  // _DSM: Device-Specific Method
                 {
-                    Local0 = Package (0x00) {}
+                    Local0 = Package (0x00){}
                     DTGP (Arg0, Arg1, Arg2, Arg3, RefOf (Local0))
                     Return (Local0)
                 }
             }
 
-            Scope (HS02) // not used
+            Scope (HS02)
             {
                 Method (_STA, 0, NotSerialized)  // _STA: Status
                 {
                     Return (Zero)
                 }
             }
 
-            Scope (HS03) // back USB A
+            Scope (HS03)
             {
                 Method (_DSM, 4, NotSerialized)  // _DSM: Device-Specific Method
                 {
-                    Local0 = Package (0x00) {}
+                    Local0 = Package (0x00){}
                     DTGP (Arg0, Arg1, Arg2, Arg3, RefOf (Local0))
                     Return (Local0)
                 }
             }
 
-            Scope (HS04) // back USB A
+            Scope (HS04)
             {
                 Method (_DSM, 4, NotSerialized)  // _DSM: Device-Specific Method
                 {
-                    Local0 = Package (0x00) {}
+                    Local0 = Package (0x00){}
                     DTGP (Arg0, Arg1, Arg2, Arg3, RefOf (Local0))
                     Return (Local0)
                 }
             }
 
-            Scope (HS05) // not used
+            Scope (HS05)
             {
                 Method (_STA, 0, NotSerialized)  // _STA: Status
                 {
                     Return (Zero)
                 }
             }
 
-            Scope (HS06) // not used
+            Scope (HS06)
             {
                 Method (_STA, 0, NotSerialized)  // _STA: Status
                 {
                     Return (Zero)
                 }
             }
 
-            Scope (HS07) // not used
+            Scope (HS07)
             {
-                Method (_STA, 0, NotSerialized)  // _STA: Status
+                Method (_DSM, 4, NotSerialized)  // _DSM: Device-Specific Method
                 {
-                    Return (Zero)
+                    Local0 = Package (0x00){}
+                    DTGP (Arg0, Arg1, Arg2, Arg3, RefOf (Local0))
+                    Return (Local0)
                 }
             }
 
-            Scope (HS08) // not used
+            Scope (HS08)
             {
-                Method (_STA, 0, NotSerialized)  // _STA: Status
+                Method (_DSM, 4, NotSerialized)  // _DSM: Device-Specific Method
                 {
-                    Return (Zero)
+                    Local0 = Package (0x00){}
+                    DTGP (Arg0, Arg1, Arg2, Arg3, RefOf (Local0))
+                    Return (Local0)
                 }
             }
```

Comment:
this does not work; this causes HS07 and HS08 to be seen as USB3.0 ports and not recognize anything being plugged in...